ui/src/app/base/components/SubnetSelect/SubnetSelect.tsx

```diff
@@ -38,7 +42,11 @@ export const SubnetSelect = ({
     subnets = subnets.filter((subnet) => subnet.vlan === vlan);
   }
 
-  const subnetOptions = subnets.map((subnet) => ({
+  if (subnets && filterFunction) {
```

Comment:
@petermakowski it turns out this prop existed in the type, but wasn't implemented.